keras_nlp/src/models/resnet/resnet_backbone.py

```diff
@@ -86,7 +86,7 @@ class ResNetBackbone(Backbone):
         stackwise_num_blocks=[2, 2, 2],
         stackwise_num_strides=[1, 2, 2],
         block_type="basic_block",
-        use_pre_activation=True,
+        version="v2",
         pooling="avg",
```

Comment:
lets keep the `use_pre_activation` arg. instead of `version` lets add arg something like `use_vd_pooling` or something like that

Reply:
Sure, I can change this.
Just a note - with two booleans, we convey that you can set `use_pre_activation = use_vd_pooling = True`. Since both architecture variants introduce subtle modifications, it will not be entirely clear from the interface what the outcome would be. My personal preference would be having it inherently mutually exclusive.

Reply:
yes, we should have two different args!